src/core/tests/type_prop/clamp.cpp

```diff
@@ -49,6 +49,19 @@ TEST(type_prop, clamp_shape_dynamic) {
     ASSERT_EQ(clamp->get_output_partial_shape(0), (PartialShape::dynamic()));
 }
 
+TEST(type_prop, clamp_evaluate_bounds) {
```

Comment:
Add test check label propagation for shape inference and via evaluate label?

Reply:
since this is Unary operator -- its input shape is just copied to the output. From this perspective label propagation during shape inference of Clamp is really simple and I don't want to spam our test library with unnecessary tests.
From the perspective of label propagation during partial value inference -- it is not even enabled -- because it isn't a real world scenario.